cmd/release-controller/round_robin.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"errors"
+	"sync"
+)
+
+var ErrEmptyItems = errors.New("cannot create round robin with no items")
+
+type RoundRobin struct {
```

Comment:
are you attached to round robin over randomly choosing one each time?  That eliminates the lock and the potential bias towards the first element and the unit tests, rigth?